✨Shader Effects✨

LlednaK opened the pull request:
Added the capacity to add json driven visuals to effects...

LlednaK committed:
Added shader effects

petrolpark commented:
Wow! Very nice. If you want the Developer Badge, do let me know of your Minecraft username.

LlednaK commented:
There was a developer badge ?! My username is Poussin98.
Maybe the code needs a little more testing…

LlednaK committed:
Changed shader mixing system to a bezier curve and removed the forgotten debug logs

LlednaK commented:
I basically changed the way that the shaders mixes with the screen by replacing the chunky and unreadable modified sine function by a slightly less not user friendly bezier curve. I also removed some hidden debug console logs and added a stack trace because "shader failed to apply O-O" doesn't help much...

LlednaK committed:
Merge branch 'Petrolpark-Mods:1.20.1' into Final

LlednaK committed:
Merge branch '1.20.1' into Final

LlednaK committed:
Obvious Fix

LlednaK committed:
Semi-Revert change (now works)

petrolpark commented:
You should have received that Badge. Can I ask that you make a PR for these changes to the [library](https://github.com/petrolpark/Create-Library)? Some of the Mob Effects in Destroy will be moved to that and it will also allow me (and other developers) to use these features in other projects. Thanks.

LlednaK commented:
I’ll have to tweak a few things and the examples I added won’t be transferred but a new PR should be done before the end of next week
